\documentclass[12pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Selective DNA and Protein Isolation from Marine Macrophyte Surfaces},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\providecommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}

  \title{\textbf{Selective DNA and Protein Isolation from Marine Macrophyte
Surfaces}}
    \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
    \author{}
    \preauthor{}\postauthor{}
    \date{}
    \predate{}\postdate{}
  
\usepackage{times} % Times New Roman font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{indentfirst}

\usepackage[labelsep=period, labelfont=bf]{caption}
\renewcommand{\thefigure}{S\arabic{figure}}
\captionsetup{justification=raggedright,singlelinecheck=false}

\usepackage{pdflscape}
\newcommand{\blandscape}{\begin{landscape}}
\newcommand{\elandscape}{\end{landscape}}

\begin{document}
\maketitle

\vspace{80mm}

\textsuperscript{1\(\dagger\)}

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:marino.korlevic@irb.hr}{\nolinkurl{marino.korlevic@irb.hr}}

1. Ruđer Bošković Institute, Center for Marine Research, G. Paliaga 5,
Rovinj, Croatia

2. University of Vienna, Department of Limnology and Bio-Oceanography,
Althanstraße 14, Vienna, Austria \newpage
\linenumbers
\setlength\parindent{24pt}

\subsection{Abstract}\label{abstract}

\newpage

\subsection{Introduction}\label{introduction}

Surfaces of marine macrophytes are inhabited by a diverse microbial
community whose structure and function are poorly understood (Egan
\emph{et al.}, 2013). As less than 1 \% of all prokaryotic species are
culturable, to study these organisms, molecular methods such as 16S rRNA
sequencing, metagenomics and metaproteomics are indispensable (Amann
\emph{et al.}, 1995; Su \emph{et al.}, 2012). Applying these techniques
requires an inital isolation step, with the purpose of obataining high
quality DNA and proteins.

Biological material (i.e.~proteins and DNA) from pelagic microbial
communities is usually isolated by collecting cells onto filters and
subsequently isolating the aimed compound (Gilbert \emph{et al.}, 2009).
If a microbial size fraction is aimed sequential filtration is applied
(Massana \emph{et al.}, 1997; Andersson \emph{et al.}, 2010). In
contrast, obtaining biological materials from microorganism inhabiting
surfaces requires either a cell detachment procedure prior to isolation
or the host material is coextracted together with the targeted material.
Methods for separating microbial cells form the host include shaking of
host tissue (Gross \emph{et al.}, 2003; Nõges \emph{et al.}, 2010),
scraping of macrophyte surfaces (Uku \emph{et al.}, 2007) or the
application of ultrasonication (Weidner \emph{et al.}, 1996; Cai
\emph{et al.}, 2014). It was shown that shaking alone is not sufficient
to remove microbial cells, at least from plant root surfaces
(Richter-Heitmann \emph{et al.}, 2016). Manual separation methods, such
as scraping and brushing, are time consuming and subjective, as the
detachment efficency depends on host tissue and the person performing
the procedure (Cai \emph{et al.}, 2014). Ultrasonication was proposed as
an alternative method as it is providing better results in terms of
detachemnt efficency (Cai \emph{et al.}, 2014; Richter-Heitmann \emph{et
al.}, 2016). The downside of this procedure is that complete cell
removal was still not obtained and tissue disruption was observed
especially after the application of probe ultrasonication
(Richter-Heitmann \emph{et al.}, 2016). An alterantive to these cell
detachment procedures is the isolation of targeted epiphytic compounds
togehter with host materials (Staufenberger \emph{et al.}, 2008; Jiang
\emph{et al.}, 2015). This procedure can lead to problems in the
following processing steps such as mitochondrial and chloroplast 16S
rRNA sequence contaminations from the host (Longford \emph{et al.},
2007; Staufenberger \emph{et al.}, 2008). In additon, when performing
metagenomics and metaproteomics host material can cause biased results
towards more abundand host DNA and proteins.

An alterantive to these procedures is a direct isolation of the targeted
material by incubating macrophyte tissues in an extraction buffer. After
the incubation is done the undistrupted host tissue is removed and the
isolation procedure continues ommiting host material contaminations. To
our knowledge the only procedure describing a direct and selective
epiphytic DNA isolation from the surfaces of marine macrophytes was
provided by Burke \emph{et al.} (2009). In contrast to previously
described methods this protocol enables an almost complete removal of
the surface community and was used for 16S rRNA gene clone libraries
construction (Burke \emph{et al.}, 2011b) and metagenomes sequencing
(Burke \emph{et al.}, 2011a). This method, thought providing a selctive
isolation procedure, is using in the extraction buffer a rapid
multienzyme cleaner (3M) that is not worldwide availabe and whose
composition is not know (Burke \emph{et al.}, 2009). Also to our
knowledge, no selective isolation protocol for proteins from epihytic
communities inhabiting marine macrophytes was established.

In the present study, we adapted a protocol widely used for DNA
isolation from filters (Massana \emph{et al.}, 1997) and a protocol used
for protein isolation from sediments (Chourey \emph{et al.}, 2010) for
selective extractions of DNA and proteins from epiphytic communities
inhabiting the surfaces of two marine macrophytes: the seagrass
\emph{Cymodocea nodosa} and the alga \emph{Caulerpa cylindracea}. In
addition, we tested the removal efficiency of the protocol and the
suitability of obtained DNA and proteins for 16S rRNA sequencing and
metaproteomics.

\newpage

\subsection{Materials and Methods}\label{materials-and-methods}

\subsubsection{Sampling}\label{sampling}

Leaves of \emph{Cymodocea nodosa} were sampled in a \emph{Cymodocea
nodosa} meadow in the Bay of Saline (45°7´5˝N; 13°37´20˝E) and in a
\emph{Cymodocea nodosa} meadow invaded by \emph{Caulerpa cylindracea} in
the proximity of the village of Funtana (45°10´39˝N 13°35´42˝E). Thalli
of \emph{Caulerpa cylindracea} were sampled in the same \emph{Cymodocea
nodosa} invaded meadow in Funtana and on a locality of only
\emph{Caulerpa cylindracea} located close to the invaded meadow.
Seagrasses and algae were collected the same day in two contrasting
seasons, on 4 December 2017 and 18 June 2018. During spring 2018 the
\emph{Cymodocea nodosa} meadow in the Bay of Saline decayed to the
extent that no leves could be retrieved (Najdek \emph{et al.},
unpublished data). Leaves and thalli were collected by diving and
transported to the laboratory in containers placed on ice and filled
with site seawater. Upon arrival to the laboratory, \emph{Cymodocea
nodosa} leaves were cut into sections of 1 -- 2 cm, while \emph{Caulerpa
cylindracea} thalli were cut into 5 -- 8 cm long sections. Leaves and
thalli were washed three times with sterile artificial seawater (ASW) to
remove loosely attached microbial cells.

\subsubsection{DNA isolation}\label{dna-isolation}

The DNA was isolated according to the protocol for isolation from
filters described in Massana \emph{et al.} (1997). This protocol was
modified and adapted for DNA isolation from microbial communites from
macrophytes surfaces as described below. 1 g wet weight of leaves and 2
g wet-weight of thalli were placed into 5 ml of lysis buffer (40 mM
EDTA, 50 mM Tris-HCl, 0.75 M sucrose; pH 8.3). Lysozyme was added (final
concentration 1 mg ml\textsuperscript{-1}) and the mixture was incubated
at 37 °C for 30 minutes. Subsequently, proteinase K (final concentration
0.5 mg ml\textsuperscript{-1}) and SDS (finl concentration 1 \%) were
added and the samples were incubated at 55 °C for 2 hours. Following the
incubation, tubes were vortexed for 10 minutes and the mixture
containign lyzed cells was separated from host leaves or thalli by
transferrring the solution into a clean tube. The lysate was extracted
twice with a mixture of phenol:chloroform:isoamyl alcohol (25:24:1; pH
8) and once with chloroform-isoamyl alcohol (24:1). After each organic
solvent mixture addition tubes were slightly vortexed and centrifugated
at 4,500 x g for 10 minutes. Following each centrifugation aqueous
phases were retrieved. After the final extraction 1/10 of 3 M sodium
acetate (ph 5.2) was added. DNA was precipitated by adding 1 volumne of
isopropanol and incubating the mixtures at -20 °C overnight.

\subsubsection{Protein isolation}\label{protein-isolation}

\subsection{Results and Discussion}\label{results-and-discussion}

\subsection{Conclusions}\label{conclusions}

\subsection{Materials and Methods}\label{materials-and-methods-1}

\newpage

\newpage

\subsection*{References}\label{references}
\addcontentsline{toc}{subsection}{References}

\hypertarget{refs}{}
\hypertarget{ref-Amann1995}{}
Amann, R.I., Ludwig, W., and Schleifer, K.H. (1995) Phylogenetic
identification and in situ detection of individual microbial cells
without cultivation. \emph{Microbiological reviews} \textbf{59}:
143--169.

\hypertarget{ref-Andersson2010}{}
Andersson, A.F., Riemann, L., and Bertilsson, S. (2010) Pyrosequencing
reveals contrasting seasonal dynamics of taxa within Baltic Sea
bacterioplankton communities. \emph{The ISME journal} \textbf{4}:
171--181.

\hypertarget{ref-Burke2009}{}
Burke, C., Kjelleberg, S., and Thomas, T. (2009) Selective extraction of
bacterial DNA from the surfaces of macroalgae. \emph{Applied and
environmental microbiology} \textbf{75}: 252--256.

\hypertarget{ref-Burke2011a}{}
Burke, C., Steinberg, P., Rusch, D., Kjelleberg, S., and Thomas, T.
(2011a) Bacterial community assembly based on functional genes rather
than species. \emph{Proceedings of the National Academy of Sciences of
the United States of America} \textbf{108}: 14288--14293.

\hypertarget{ref-Burke2011b}{}
Burke, C., Thomas, T., Lewis, M., Steinberg, P., and Kjelleberg, S.
(2011b) Composition, uniqueness and variability of the epiphytic
bacterial community of the green alga Ulva australis. \emph{The ISME
journal} \textbf{5}: 590--600.

\hypertarget{ref-Cai2014}{}
Cai, X., Gao, G., Yang, J., Tang, X., Dai, J., Chen, D., and Song, Y.
(2014) An ultrasonic method for separation of epiphytic microbes from
freshwater submerged macrophytes. \emph{Journal of basic microbiology}
\textbf{54}: 758--761.

\hypertarget{ref-Chourey2010}{}
Chourey, K., Jansson, J., VerBerkmoes, N., Shah, M., Chavarria, K.L.,
Tom, L.M., et al. (2010) Direct Cellular Lysis/Protein Extraction
Protocol for Soil Metaproteomics. \emph{Journal of Proteome Research}
\textbf{9}: 6615--6622.

\hypertarget{ref-Egan2013}{}
Egan, S., Harder, T., Burke, C., Steinberg, P., Kjelleberg, S., and
Thomas, T. (2013) The seaweed holobiont: understanding seaweed-bacteria
interactions. \emph{FEMS microbiology reviews} \textbf{37}: 462--476.

\hypertarget{ref-Gilbert2009}{}
Gilbert, J.A., Field, D., Swift, P., Newbold, L., Oliver, A., Smyth, T.,
et al. (2009) The seasonal structure of microbial communities in the
Western English Channel. \emph{Environmental Microbiology} \textbf{11}:
3132--3139.

\hypertarget{ref-Gross2003}{}
Gross, E.M., Feldbaum, C., and Graf, A. (2003) Epiphyte biomass and
elemental composition on submersed macrophytes in shallow eutrophic
lakes. \emph{Hydrobiologia} \textbf{506-509}: 559--565.

\hypertarget{ref-Jiang2015}{}
Jiang, Y.-F., Ling, J., Dong, J.-D., Chen, B., Zhang, Y.-Y., Zhang,
Y.-Z., and Wang, Y.-S. (2015) Illumina-based analysis the microbial
diversity associated with Thalassia hemprichii in Xincun Bay, South
China Sea. \emph{Ecotoxicology (London, England)} \textbf{24}: 1548--56.

\hypertarget{ref-Longford2007}{}
Longford, S., Tujula, N., Crocetti, G., Holmes, A., Holmström, C.,
Kjelleberg, S., et al. (2007) Comparisons of diversity of bacterial
communities associated with three sessile marine eukaryotes.
\emph{Aquatic Microbial Ecology} \textbf{48}: 217--229.

\hypertarget{ref-Massana1997}{}
Massana, R., Murray, A.E., Preston, C.M., and DeLong, E.F. (1997)
Vertical distribution and phylogenetic characterization of marine
planktonic Archaea in the Santa Barbara Channel. \emph{Applied and
environmental microbiology} \textbf{63}: 50--56.

\hypertarget{ref-Noges2010}{}
Nõges, T., Luup, H., and Feldmann, T. (2010) Primary production of
aquatic macrophytes and their epiphytes in two shallow lakes (Peipsi and
Võrtsjärv) in Estonia. \emph{Aquatic Ecology} \textbf{44}: 83--92.

\hypertarget{ref-Richter-Heitmann2016}{}
Richter-Heitmann, T., Eickhorst, T., Knauth, S., Friedrich, M.W., and
Schmidt, H. (2016) Evaluation of Strategies to Separate Root-Associated
Microbial Communities: A Crucial Choice in Rhizobiome Research.
\emph{Frontiers in Microbiology} \textbf{7}: 773.

\hypertarget{ref-Staufenberger2008}{}
Staufenberger, T., Thiel, V., Wiese, J., and Imhoff, J.F. (2008)
Phylogenetic analysis of bacteria associated with Laminaria saccharina.
\emph{FEMS Microbiology Ecology} \textbf{64}: 65--77.

\hypertarget{ref-Su2012}{}
Su, C., Lei, L., Duan, Y., Zhang, K.-Q., and Yang, J. (2012)
Culture-independent methods for studying environmental microorganisms:
methods, application, and perspective. \emph{Applied microbiology and
biotechnology} \textbf{93}: 993--1003.

\hypertarget{ref-Uku2007}{}
Uku, J., Björk, M., Bergman, B., and Díez, B. (2007) Characterization
and comparison of prokaryotic epiphytes associated with three East
African seagrasses. \emph{Journal of Phycology} \textbf{43}: 768--779.

\hypertarget{ref-Weidner1996}{}
Weidner, S., Arnold, W., and Puhler, A. (1996) Diversity of uncultured
microorganisms associated with the seagrass Halophila stipulacea
estimated by restriction fragment length polymorphism analysis of
PCR-amplified 16S rRNA genes. \emph{Applied and environmental
microbiology} \textbf{62}: 766--771.


\end{document}
