---
title: "**Selective DNA and Protein Isolation from Marine Macrophyte Surfaces**"
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
fontsize: 12pt
geometry: margin=1.0in
csl: citation_style.csl #Get themes at https://github.com/citation-style-language/styles
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```

\vspace{20mm}
Marino Korlević^1$*$^, Marsej Markovski^1^, Zihao Zhao^2^, Gerhard J. Herndl^2,3^, Mirjana Najdek^1^

1\. Center for Marine Research, Ruđer Bošković Institute, Croatia

2\. Department of Functional and Evolutionary Ecology, University of Vienna, Austria

3\. Department of Marine Microbiology and Biogeochemistry, Royal Netherlands Institute for Sea Research, Utrecht University, The Netherlands

^$*$^To whom correspondence should be addressed:

Marino Korlević

G. Paliaga 5, 52210 Rovinj, Croatia

Tel.: +385 52 804 768

Fax: +385 52 804 780

e-mail: marino.korlevic@irb.hr

Running title: DNA and protein isolation from macrophyte surfaces

\newpage
\linenumbers
\sisetup{mode=text}
\setlength\parindent{24pt}

## Summary


\newpage
## Introduction
Surfaces of marine macrophytes are inhabited by a diverse microbial community whose structure and function are poorly understood [@Egan2013]. As less than 1 % of all prokaryotic species are culturable, to study these organisms, molecular methods such as 16S rRNA sequencing, metagenomics and metaproteomics are indispensable [@Amann1995; @Su2012]. Applying these techniques requires an initial isolation step, with the purpose of obtaining high quality DNA and proteins.

Biological material (i.e. proteins and DNA) from pelagic microbial communities is usually isolated by collecting cells onto filters and subsequently isolating the aimed compound [@Gilbert2009]. If a microbial size fraction is aimed sequential filtration is applied [@Massana1997; @Andersson2010]. In contrast, obtaining biological materials from microorganism inhabiting surfaces requires either a cell detachment procedure prior to isolation or the host material is coextracted together with the targeted material. Methods for separating microbial cells form the host include shaking of host tissue [@Gross2003; @Noges2010], scraping of macrophyte surfaces [@Uku2007] or the application of ultrasonication [@Weidner1996; @Cai2014]. It was shown that shaking alone is not sufficient to remove microbial cells, at least from plant root surfaces [@Richter-Heitmann2016]. Manual separation methods, such as scraping and brushing, are time consuming and subjective, as the detachment efficiency depends on host tissue and the person performing the procedure [@Cai2014]. Ultrasonication was proposed as an alternative method as it is providing better results in terms of detachment efficiency [@Cai2014; @Richter-Heitmann2016]. The downside of this procedure is that complete cell removal was still not obtained and tissue disruption was observed especially after the application of probe ultrasonication [@Richter-Heitmann2016]. An alternative to these cell detachment procedures is the isolation of targeted epiphytic compounds together with host materials [@Staufenberger2008; @Jiang2015]. This procedure can lead to problems in the following processing steps such as mitochondrial and chloroplast 16S rRNA sequence contaminations from the host [@Longford2007; @Staufenberger2008]. In addition, when performing metagenomics and metaproteomics host material can cause biased results towards more abundant host DNA and proteins.

An alternative to these procedures is a direct isolation of the targeted material by incubating macrophyte tissues in an extraction buffer. After the incubation is done the undisrupted host tissue is removed and the isolation procedure continues omitting host material contaminations. To our knowledge the only procedure describing a direct and selective epiphytic DNA isolation from the surfaces of marine macrophytes was provided by @Burke2009. In contrast to previously described methods this protocol enables an almost complete removal of the surface community and was used for 16S rRNA gene clone libraries construction [@Burke2011a] and metagenomes sequencing [@Burke2011]. This method, thought providing a selective isolation procedure, is using in the extraction buffer a rapid multienzyme cleaner (3M) that is not worldwide available and whose composition is not know [@Burke2009]. Also to our knowledge, no selective isolation protocol for proteins from epiphytic communities inhabiting marine macrophytes was established.

In the present study, we adapted a protocol widely used for DNA isolation from filters [@Massana1997] and a protocol used for protein isolation from sediments [@Chourey2010; @Hultman2015] for selective extractions of DNA and proteins from epiphytic communities inhabiting the surfaces of two marine macrophytes: the seagrass *Cymodocea nodosa* and the alga *Caulerpa cylindracea*. In addition, we tested the removal efficiency of the protocol and the suitability of obtained DNA and proteins for 16S rRNA sequencing and metaproteomics.

\newpage
## Results

```{r community}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(!str_detect(taxon, "^Eukaryota")) %>%
  filter(taxon!="Root")

# Remove Mitochondria sequences and subtract their number from higher taxonomic levels to which they belong
mitochondria <- filter(community, str_detect(taxon, "^Mitochondria$"))$rankID
community <- mutate_at(community, 5:ncol(community), list(~case_when(
    rankID==str_extract(mitochondria, "(\\d+\\.){4}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Mitochondria"],
    TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Mitochondria")) %>%
  select(-ATCC_1, -NC_1)

# Calculating the number of sequences
number_sequences <- filter(community, taxon=="Archaea" | taxon=="Bacteria") %>%
  select(total) %>%
  sum()

# Calculating the number of chloroplast sequences
number_chloroplast_sequences <- filter(community, taxon=="Chloroplast") %>%
  select(total) %>%
  sum()

# Calculating the number of sequences without chloroplast sequences
number_sequences_without_chloroplast <- number_sequences - number_chloroplast_sequences

# Calculating the proprotion of chloroplast sequences
proprotion_chloroplast_sequences <- (number_chloroplast_sequences / number_sequences) * 100

# Calculating the minimal and maximal number of sequences
min_max <- filter(community, taxon=="Archaea" | taxon=="Bacteria") %>%
  summarise_at(6:ncol(.), sum) %>%
  summarise(min=min(.), max=max(.))
```

```{r shared}
# Loading OTU/sample data
shared <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.opti_mcc.shared")

# Calculating the number of OTUs before rarefaction
no_otus <- select(shared, starts_with("Otu")) %>%
  colnames() %>%
  length()
```

```{r taxonomy}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(!str_detect(taxon, "^Eukaryota")) %>%
  filter(taxon!="Root")

# Remove Mitochondria sequences and subtract their number from higher taxonomic levels to which they belong
mitochondria <- filter(community, str_detect(taxon, "^Mitochondria$"))$rankID
community <- mutate_at(community, 5:ncol(community), list(~case_when(
    rankID==str_extract(mitochondria, "(\\d+\\.){4}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Mitochondria"],
    TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Mitochondria")) %>%
  select(-ATCC_1, -NC_1) %>%
  group_by(taxlevel) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup()

# Loading metadata of each sample and selection of desired information of each sample
metadata <- read_tsv("data/raw/metadata.csv") %>%
  filter(ID %in% colnames(community[, 6:ncol(community)])) %>%
  select(ID, label) %>%
  deframe()

# Selection of groups for plotting
plot <- filter(community, taxlevel==2 |
                 (taxlevel==4 & str_detect(taxon, "^Chloroplast$")) |
                 (taxlevel==3 & str_detect(rankID, filter(community, str_detect(taxon, "^Proteobacteria$"))[[2]]))) %>%
  filter_at(6:ncol(.), any_vars(. >= 1)) %>%
  mutate_at(5:ncol(.), list(~case_when(taxon=="Cyanobacteria" ~ . - .[taxon=="Chloroplast"], TRUE ~ .))) %>%
  mutate_at(5:ncol(.), list(~case_when(taxon=="Proteobacteria" ~ . - sum(.[taxlevel==3 & str_detect(rankID, filter(community, str_detect(taxon, "^Proteobacteria$"))[[2]])]), TRUE ~ .))) %>%
  mutate(taxon=str_replace(taxon, "Proteobacteria", "Other Proteobacteria")) %>%
  mutate(taxon=str_replace_all(taxon, c("unknown_unclassified"="No_Relative", "unknown"="No_Relative"))) %>%
  filter_at(6:ncol(.), any_vars(. >= 1)) %>%
  bind_rows(summarise_all(., list(~ifelse(is.numeric(.), 100-sum(.), paste("Other"))))) %>%
  rename_at(names(metadata), ~unname(metadata)) %>%
  arrange(taxon %in% "No_Relative") %>%
  mutate(mean=rowMeans(select(., contains("SCy"), contains("FCyM"), contains("FCaM"), contains("FCa")), na.rm=TRUE)) %>%
  mutate(sd=rowSds(as.matrix(select(., contains("SCy"), contains("FCyM"), contains("FCaM"), contains("FCa"))), na.rm=TRUE)) %>%
  arrange(desc(mean))
```

```{r chloroplast}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/chloroplast.pds.wang.tax.summary") %>%
  filter(taxon!="Root")

# Calculating relative abundances
community <- group_by(community, taxlevel) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup()

# Loading metadata of each sample and selection of desired information of each sample
metadata <- read_tsv("data/raw/metadata.csv") %>%
  filter(ID %in% colnames(community[, 6:ncol(community)])) %>%
  select(ID, label) %>%
  deframe()

# Selection of groups for plotting
chloroplast <- filter(community, taxlevel==5) %>%
  rename_at(names(metadata), ~unname(metadata)) %>%
  mutate(mean=rowMeans(select(., contains("SCy"), contains("FCyM"), contains("FCaM"), contains("FCa")), na.rm=TRUE)) %>%
  mutate(sd=rowSds(as.matrix(select(., contains("SCy"), contains("FCyM"), contains("FCaM"), contains("FCa"))), na.rm=TRUE)) %>%
  arrange(desc(mean))
```

To assess the removal efficiency of DNA and protein isolation procedures leaves and thalli were stained with SYBR Green I and examined under a confocal microscope before and after treatments were performed. As it is known that marine macrophytes are harboring more algal epiphytes during Autumn and Winter months  [@Reyes2001], the sampling was performed in two contrasting seasons, in December 2017 when higher abundances of algal epiphytes are expected, and in June 2018, when lower abundances could be expected. In addition, to further test the isolation protocols on different macrophyte species, DNA and proteins were isolated from the surfaces of two macrophytes, the seagrass *C. nodosa* and the macroalga *C. cylindracea*. *C. nodosa* was sampled in a monospecific meadow (Bay of Saline) and *C. cylindracea* invaded meadow (Bay of Funtana), while samples of *C. cylindracea* originated from a monospecific settlement and a *C. cylindracea* invaded *C. nodosa* meadow in the Bay of Funtana. *C. nodosa* June samples from the Bay of Saline were missing as the meadow went to a decline from Spring 2018 (M. Najdek, personal communication). Procedures developed for DNA and protein isolation showed an almost complete removal of the surface community on both, samples collected in December 2017 (\autoref{confocal_december}) and June 2018 (\autoref{confocal_june}). In addition, a similar removal efficiency was observed for communities associated with *C. nodosa* and *C. cylindracea*. Also, no effect of station, settlement or isolation procedure (DNA or protein) on the removal efficiency was observed (Figs. \ref{confocal_december} and \ref{confocal_june}).

To evaluate if the obtained DNA is suitable to determine the microbial community structure an Illumina sequencing of the V4 16S rRNA region was performed. Sequencing yielded a total of `r number_sequences` sequences after quality curation and exclusion of eukaryotic, mitochondrial and no relative sequences. The number of sequences classified as chloroplast was `r number_chloroplast_sequences` (`r proprotion_chloroplast_sequences` \si{\percent})˛ After the exclusion of these sequences the total number of retrieved reads dropped to `r number_sequences_without_chloroplast`, ranging from `r min_max$min` and `r min_max$max` sequences per sample (\autoref{supp-nseq_notus}). Even when the highest sequencing effort was applied the rarefaction curves did not level off that is commonly observed in high-throughput 16S rRNA amplicon sequencing procedures (\autoref{supp-rarefaction}). Sequence clustering at a similarity level of 97 \si{\percent} yielded a total of `r no_otus` different OTUs. Taxonomic classification of reads allowed the determination of the macrophyte associated epiphytic community that was mainly composed of: *Alphaproteobacteria* (`r filter(plot, taxon=="Alphaproteobacteria")$mean` ± `r filter(plot, taxon=="Alphaproteobacteria")$sd` \si{\percent}), *Bacteroidota* (`r filter(plot, taxon=="Bacteroidota")$mean` ± `r filter(plot, taxon=="Bacteroidota")$sd` \si{\percent}), *Gammaproteobacteria* (`r filter(plot, taxon=="Gammaproteobacteria")$mean` ± `r filter(plot, taxon=="Gammaproteobacteria")$sd` \si{\percent}), *Desulfobacterota* (`r filter(plot, taxon=="Desulfobacterota")$mean` ± `r filter(plot, taxon=="Desulfobacterota")$sd` \si{\percent}), *Cyanobacteria* (`r filter(plot, taxon=="Cyanobacteria")$mean` ± `r filter(plot, taxon=="Cyanobacteria")$sd` \si{\percent}) and *Planctomycetota* (`r filter(plot, taxon=="Planctomycetota")$mean` ± `r filter(plot, taxon=="Planctomycetota")$sd` \si{\percent}) (\autoref{community}).

Primers used in this study, as in many other 16S rRNA amplicon procedures, also amplify chloroplast 16S rRNA genes. We observed a high proportion of chloroplast sequences in all analyzed samples (`r filter(plot, taxon=="Chloroplast")$mean` ± `r filter(plot, taxon=="Chloroplast")$sd` \si{\percent}) (\autoref{community}). To determine if chloroplast sequences originate from hosts or eukaryotic epiphytic organisms, we exported SILVA classified chloroplast sequences and reclassified them using the RDP (Ribosomal Database Project) training set that allows for a more detailed chloroplast classification. The largest proportion of sequences was classified as Bacillariophyta (`r filter(chloroplast, taxon=="Bacillariophyta")$mean` ± `r filter(chloroplast, taxon=="Bacillariophyta")$sd` \si{\percent}) indicating that the DNA removal procedure did not coextract larger quantities of host DNA (\autoref{chloroplast}).



## Discussion

\newpage
## Experimental procedures
### Sampling
Leaves of *C. nodosa* were sampled in a *C. nodosa* meadow in the Bay of Saline (\ang{45;7;5} N, \ang{13;37;20} E) and in a *C. nodosa* meadow invaded by *C. cylindracea* in the proximity of the village of Funtana (\ang{45;10;39} N, \ang{13;35;42} E). Thalli of *C. cylindracea* were sampled in the same *C. nodosa* invaded meadow in Funtana and on a locality of only *C. cylindracea* located in the proximity of the invaded meadow. Leaves and thalli were collected on the same day in two contrasting seasons, on 4 December 2017 and 18 June 2018. During spring 2018 the *C. nodosa* meadow in the Bay of Saline decayed to an extent that no leaves could be retrieved (Najdek *et al.*, unpublished data). Leaves and thalli were collected by diving and transported to the laboratory in containers placed on ice and filled with site seawater. Upon arrival to the laboratory, *C. nodosa* leaves were cut into sections of 1 -- 2 \si{\cm}, while *C. cylindracea* thalli were cut into 5 -- 8 \si{\cm} long sections. Leaves and thalli were washed three times with sterile artificial seawater (ASW) to remove loosely attached microbial cells.

### DNA isolation
The DNA was isolated according to the protocol for isolation from filters described in @Massana1997. This protocol was modified and adapted for DNA isolation from microbial communities from macrophyte surfaces as described below. 5 \si{\ml} of lysis buffer (40 \si{\milli\Molar} EDTA, 50 \si{\milli\Molar} Tris-HCl, 0.75 \si{\Molar} sucrose; pH 8.3) was added to 1 \si{\g} wet weight of leaves or \si{\g} wet-weight of thalli. Lysozyme was added (final concentration 1 \si{\mg\per\ml}) and the mixture was incubated at 37 \si{\degreeCelsius} for 30 \si{\minute}. Subsequently, proteinase K (final concentration 0.5 \si{\mg\per\ml}) and SDS (final concentration 1 \si{\percent}) were added and the samples were incubated at 55 \si{\degreeCelsius} for 2 \si{\hour}. Following the incubation, tubes were vortexed for 10 \si{\minute} and the mixture containing lyzed epiphytic cells was separated from host leaves or thalli by transferring the solution into a clean tube. The lysate was extracted twice with a mixture of phenol:chloroform:isoamyl alcohol (25:24:1; pH 8) and once with chloroform:isoamyl alcohol (24:1). After each organic solvent mixture addition tubes were slightly vortexed and centrifuged at 4,500 × g for 10 \si{\minute}. Following each centrifugation aqueous phases were retrieved. After the final extraction 1/10 of chilled 3 \si{\Molar} sodium acetate (pH 5.2) was added. DNA was precipitated by adding 1 volume of chilled isopropanol, incubating the mixtures overnight at \num{-20} °C and centrifuging at 16,000 × g and 4 \si{\degreeCelsius} for 20 \si{\minute}. The pellet was washed twice with 1 \si{\ml} of chilled 70 \si{\percent} ethanol and centrifuged after each washing step at 20,000 × g and 4 \si{\degreeCelsius} for 10 \si{\minute}. After the first washing step duplicate pellets form the same sample were pooled and transferred to a clean 1.5 \si{\ml} tube. The dried pellet was resuspended in 100 \si{\ul} of deionized water.

### Illumina 16S rRNA sequencing
An aliquot of isolated DNA was treated with RNase A (final concentration 200 \si{\ug\per\ml}) for 2 \si{\hour} at 37 \si{\degreeCelsius}. The DNA concentration was determined using the Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen, USA) according to the manufacturer’s instructions and diluted to 1 \si{\ng\per\ul}. The V4 region of the 16S rRNA gene was amplified using a two-step PCR procedure. In the first PCR the 515F (5'-GTGYCAGCMGCCGCGGTAA-3') and 806R (5'-GGACTACNVGGGTWTCTAAT-3') primers from the Earth Microbiome Project (http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/16s/) were used to amplify the target region [@Caporaso2012; @Apprill2015; @Parada2016]. These primers contained on their 5' end a tagged sequence. Each sample was amplified in four parallel 25 \si{\ul} reactions of which each contained: 1 × Q5 Reaction Buffer , 0.2 \si{\milli\Molar} of dNTPmix, 0.7 \si{\mg\per\ml} BSA (Bovine Serum Albumin), 0.2 \si{\micro\Molar} of forward and reverse primers, 0.5 U of Q5 High-Fidelity DNA Polymerase (New England Biolabs, USA) and 5 \si{\ng} of DNA template. Cycling conditions were: initial denaturation at 94 \si{\degreeCelsius} for 3 \si{\minute}, 20 cycles of denaturation at 94 \si{\degreeCelsius} for 45 \si{\s}, annealing at 50 \si{\degreeCelsius} for 60 \si{\s} and elongation at 72 \si{\degreeCelsius} for 90 \si{\s}, finalized by an elongation step at 72 \si{\degreeCelsius} for 10 \si{\minute}. The four parallel reactions volumes were pooled and PCR products were purified using the GeneJET PCR Purification Kit (Thermo Fisher Scientific, USA) according to the manufacturer's instructions and following the protocol that included isopropanol addition for better small DNA fragment yield. The column was eluted in 30 \si{\ul} of deionized water. Purified PCR products were sent for Illumina MiSeq sequencing (2 × 250 bp) at IMGM Laboratories, Martinsried, Germany. Before sequencing, at IMGM the second PCR amplification of the two-step PCR procedure was performed using primers targeting the tagged region incorporated in the first PCR. In addition, these primers contained adapter and sample-specific index sequences. The second PCR was carried out for 8 cycles. Beside samples, a positive and negative control were sequenced. A negative control was comprised of four parallel PCR reactions without DNA template, while for a positive control a mock community composed of evenly mixed DNA material originating from 20 bacterial strains (ATCC MSA-1002, ATCC, USA) was used. Sequences obtained in this study have been deposited in the European Nucleotide Archive (ENA) at EMBL-EBI under accession numbers SAMEA6786270, SAMEA6648792 -- SAMEA6648794, SAMEA6648809 -- SAMEA6648811.

```{r}
# Calculation of the error rate based on the ATCC MSA-1002 mock community
error <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.error.summary", na="") %>%
  filter(numparents==1) %>%
  mutate(mismatches_weight=weight*mismatches) %>%
  mutate(total_weight=weight*total)
error <- sum(error$mismatches_weight)/sum(error$total_weight)*100
# Calculation of the number of sequences in the negative control
nc <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(taxon=="Root") %>%
  select(NC_1)
```
Obtained sequences were analyzed on the computer cluster Isabella (University Computing Center, University of Zagreb) using mothur (version 1.43.0) [@Schloss2009] according to the MiSeq Standard Operating Procedure (MiSeq SOP; https://mothur.org/wiki/MiSeq_SOP) [@Kozich2013] and recommendations given from the Riffomonas project to enhance data reproducibility (http://www.riffomonas.org/). For alignment and classification of sequences the SILVA SSU Ref NR 99 database (release 138; http://www.arb-silva.de) was used [@Quast2013; @Yilmaz2014]. Sequences classified as chloroplasts by SILVA were exported and reclassifed using mothur and the RDP (http://rdp.cme.msu.edu/) training set (version 16) reference files adapted for mothur [@Cole2014]. In comparison to SILVA, RDP allows a more detailed classification of chloroplast sequences. Pipeline data processing and visualization was done using R (version 3.6.0) [@RCoreTeam2019],  package tidyverse (version 1.3.0) [@Wickham2019] and multiple other packages [@Xie2014; @Xie2015; @Xie2019a; @Xie2019; @Xie2020; @Allaire2019; @Xie2018; @Zhu2019; @Neuwirth2014; @Bengtsson2020]. The detailed analysis procedure including the R Markdown file for this paper are available as a GitHub repository (https://github.com/mkorlevic/Korlevic_SelectiveRemoval_EnvironMicrobiol_2020). Based on the ATCC MSA-1002 mock community included in the analysis a sequencing error rate of `r error` \si{\percent} was determined, which is in line with previously reported values for next-generation sequencing data [@Kozich2013; @Schloss2016]. In addition, the negative control processed together with the samples yielded only `r nc` sequences after sequence quality curation.

### Metagenomics
Four samples were selected and sent on dry ice to IMGM Laboratories, Martinsried, Germany for metagenomic sequencing. DNA was purified using AMPure XP Beads (Beckman Coulter, USA) applying a bead:DNA ratio of 1:1 (v/v), quantified with a Qubit dsDNA HS Assay Kit (Thermo Fisher Scientific, USA) and check for integrity on a 1 \si{\percent} agarose gel. Metagenomic sequencing libraries were generated from 300 \si{\ng} of input DNA using a NEBNext Ultra II FS DNA Library Prep Kit for Illumina (New England Biolabs, USA) according to the manufacturer's instructions. Fragments were selected (500 -- 700 bp) using AMPure XP Beads, PCR enriched for 3 -- 5 cycles and quality controlled. Libraries generated from different DNA samples were pooled and sequenced on an Illumina NovaSeq 6000 sequencing system (2 × 250 bp).

Obtained sequences were assembled individually using MEGAHIT (version  1.1.2) [@Li2015] under default settings. Putative genes were predicted from contings longer than 200 bp using Prodigal (version  2.6.3) [@Hyatt2010] in metagenome mode (-p meta). Abundances of predicted genes were expressed as Reads Per Kilobase Million (RPKM) and calculated using the BWA algorithm (version  0.7.16a) [@Li2009a]. All predicted genes were functionally annotated using the eggNOG-mapper [@Huerta-Cepas2017] and eggNOG database (version 5.0) [@Huerta-Cepas2019]. The taxonomic classification of sequences was determined using the lowest common ancestor algorithm adapted from DIAMOND (version 0.8.36) [@Buchfink2015] and by searching against the non-redundant database (NR). To determine phylogeny the top 10 \si{\percent} hits with an e-value < 1 × 10^-5^ were used (\-\-top 10).

### Protein isolation
Proteins were isolated according to the protocol for isolation from soil described in @Chourey2010 and modified by @Hultman2015. These protocols were further modified and adapted for protein isolation from microbial communities form macrophyte surfaces as described below. 20 \si{\ml} of protein extraction buffer (4 \si{\percent} SDS, 100 \si{\milli\Molar} Tris-HCl [pH 8.0]) was added to 5 \si{\g} wet weight of leaves or 10 \si{\g} wet weight of thalli. The mixture was incubated in boiling water for 5 \si{\minute}, vortexed for 10 \si{\minute} and incubated again in boiling water for 5 \si{\minute}. After a brief vortex the lysate was transferred to a clean tube separating the host leaves or thalli from the mixture containing lyzed epiphytic cells. Dithiothreitol (DTT; final concentration 24 \si{\milli\Molar}) was added and proteins were precipitated with chilled 100 \si{\percent} trichloroacetic acid (TCA; final concentration 20 \si{\percent}) overnight at \num{-20} °C. Precipitated proteins were centrifuged at 10,000 × g and 4 \si{\degreeCelsius} for 40 \si{\minute}. The obtained protein pellet was washed three times with chilled acetone. During the first washing step the pellet was transferred to a clean 1.5 \si{\ml} tube. After each washing step samples were centrifuged at 20,000 × g and 4 \si{\degreeCelsius} for 5 \si{\minute}. Dried pellets were stored at \num{-80} °C until further analysis.

### Metaproteomics
Isolated proteins were whole trypsin digested using the FASP (filter-aided sample preparation) Protein Digestion Kit (Expedeon, UK) according to the manufacturer's instructions [@Wisniewski2009] with small modifications. Before loading the solution to the column, protein pellets were solubilized in a urea sample buffer included in the kit amended with DTT (final concentration 100 \si{\milli\Molar}) for 45 \si{\minute} at room temperature and centrifuged at 20,000 × g for 2 -- 5 \si{\minute} at room temperature to remove larger particles. The first washing step after protein solution loading was repeated twice. In addition, centrifugation steps were prolonged if the column was clogged. Trypsin digestion was performed on column filters at 37 \si{\degreeCelsius} overnight for 18 \si{\hour}. The final filtrate containing digested proteins was acidified with 1 \si{\percent} (final concentration) trifluoroacetic acid, freezed at \num{-80} \si{\degreeCelsius} for 15 \si{\minute}, lyophilized and sent to the Vienna Metabolomics Center (University of Vienna) for metaproteomic analysis. Peptides were resuspended in 1 \si{\percent} (final concentration) trifluoroacetic acid, desalted using the Pierce C18 Tips (Thermo Fisher Scientific, USA) according to the manufacturer's instructions and sequenced on a Q Exactive Hybrid Quadrupole-Orbitrap Mass Spectrometer (Thermo Fisher Scientific, USA). Obtained MS/MS spectra were searched against a protein database from metagenomic assembly published in @Burke2011 using SEQUEST-HT engines and validated with Percolator in Proteome Discoverer 2.1 (Thermo Fisher Scientific, USA). The target-decoy approach was used to reduce the probability of false peptide identification. Results whose false discovery rate at the peptide level was \SI{< 1}{\percent} were kept. For protein identification a minimum of of two peptides and one unique peptide were required. For protein quantification, a chromatographic peak area-based free quantitative method was applied.

### Confocal microscopy
Host leaves and thalli from DNA and protein isolation steps were washed seven times in deionized water and fixed with formaldehyde (final concentration ~ 3 \si{\percent}). In addition, nontreated leaves and thalli, washed three times in ASW to remove loosely attached microbial cells, were fixed in the same concentration of formaldehyde and used as a positive control. For long therm storage, fixed leaves and thalli were immersed in a mixture of phosphate-buffered saline (PBS) and ethanol (1:1) and stored at \num{-20} \si{\degreeCelsius}. Treated and untreated leaves and thalli segments were stained in a 2 × solution of SYBR Green I and examined under a Leica TCS SP8 X FLIM confocal microscope (Leica Microsystems, Germany).

## Acknowledgements

\newpage
## References
<div id="refs"></div>

\newpage 
\setlength\parindent{0pt}

## Figure Captions
**\autoref{confocal_december}.** \nameref{confocal_december}

**\autoref{confocal_june}.** \nameref{confocal_june}

**\autoref{community}.** \nameref{community}

**\autoref{chloroplast}.** \nameref{chloroplast}

\newpage
## Figures
```{r out.width="85%", fig.align="center", fig.cap="Confocal microscope images of \\textit{C. nodosa} and \\textit{C. cylindracea} surfaces from the Bay of Saline and the Bay of Funtana (Mixed and Monospecific Settlements) sampled on 4 December 2017 and stained with SYBR Green I. Scale bar in all images is 60 \\si{\\um}.\\label{confocal_december}", fig.pos="ht"}
include_graphics("../results/images/confocal_december.jpg")
```

\newpage
```{r out.width="85%", fig.align="center", fig.cap="Confocal microscope images of \\textit{C. nodosa} and \\textit{C. cylindracea} surfaces from the Bay of Funtana (Mixed and Monospecific Settlements) sampled on 19 June 2018 and stained with SYBR Green I. Scale bar in all images is 60 \\si{\\um}.\\label{confocal_june}", fig.pos="ht"}
include_graphics("../results/images/confocal_june.jpg")
```

\newpage
```{r out.width="100%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant (> 1 \\si{\\percent}) bacterial and archaeal sequences from surfaces of two marine macrophytes (\\textit{C. nodosa} and \\textit{C. cylindracea}) sampled in the Bay of Saline and the Bay of Funtana (Mixed and Monospecific Settlements) and in two contrasting seasons (4 December 2017 and 19 June 2018).\\label{community}", fig.pos="ht"}
include_graphics("../results/figures/community_bar_plot.jpg")
```

\newpage
```{r out.width="100%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of chloroplast sequences from surfaces of two marine macrophytes (\\textit{C. nodosa} and \\textit{C. cylindracea}) sampled in the Bay of Saline and the Bay of Funtana (Mixed and Monospecific Settlements) and in two contrasting seasons (4 December 2017 and 19 June 2018).\\label{chloroplast}", fig.pos="ht"}
include_graphics("../results/figures/chloroplast.jpg")
```